app/scripts/templates/sign_in_unblock.mustache

```diff
@@ -0,0 +1,43 @@
+<div id="main-content" class="card">
+  {{#error}}
+    <header>
+      <h1 id="fxa-signin-unblock-header">{{#t}}Authorize this sign-in{{/t}}</h1>
```

Comment:
this header is in both places for `#error`, can we just move it out of the condition?